### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by andreabettich**
image
image

### CODE REVIEW

**Comment by andreabettich** [comment]
resolve #9

File: public/index.html [hunk 1]
```diff
@@ -16,18 +16,30 @@
...
+              <li class="language__item"><a href="#">NL</a></li>
+              <li class="language__item"><a href="#">EN</a></li>
+              <li class="language__item"><a href="#">FR</a></li>
+            </ul>
+          </li>
+          <li class="menu__item">
+            <label for="search">
+              <i class="navigation__search fa fa-search" aria-hidden="true"></i>
+            </label>
+            <input class="search__toggle" class type="checkbox" id="search" />
```
Comment (kennyzehn) [inline public/index.html]: there are two class attributes?! :P

File: public/index.html [hunk 2]
```diff
@@ -16,18 +16,30 @@
           <li class="menu__item"><a href="#">Kandidaten</a></li>
           <li class="menu__item"><a href="#">Vergleichen</a></li>
           <li class="menu__item"><a href="#">Partner</a></li>
+          <li class="menu__item"><i class="navigation__divider"></i></li>
+          <li class="menu__item">
+            <label for="language">
+              DE<i class="fa fa-chevron-down" aria-hidden="true"></i>
+            </label>
+            <input class="language__toggle" class type="checkbox" id="language" />
```
Comment (kennyzehn) [inline public/index.html]: there are two class attributes?! :P

Comment (andreabettich) [reply public/index.html]: fixed

File: public/index.html (hunk 1 above)
Comment (andreabettich) [reply public/index.html]: fixed

**Pull request merged**